explainer.md

```diff
@@ -0,0 +1,66 @@
+The second option gives a more usable way to the developers since if their code is capable of handling a `PointerEvent` object already they could easily just pass the coalesced events to that and it would work fine.
+
+Because of the ease of use for the developers and also after getting some feedback from the developer, the working group chose the second option.
+You can fine examples on how to use the proposed API in the [proposed specification](https://w3c.github.io/pointerevents/extension.html#examples).
+
+
+## Predicted Events
+This is the proposal to match other platforms predicted events API for the events to reduce the general user perception of latency.
+
+### Problem
+Imagine a drawing app that draws the points as it gets from an input device such as mouse.
+There is always some amount of latency compare to when the users draw on a paper due to inherent UA input pipeline latency and graphics pipeline latency and buffering.
+So having some sort of the prediction to draw preemptively a little in the future would reduce the perception of the latency. Having the API as part of the web platform enables UA to use some of the underlying platform APIs as well as using other attributes that might not be exposed to the web directly in the prediction algorithm.
+
+### Solution
+We propose to provide an API called `getPredictedEvents` on `PointerEvent` object. This API format will have the same concerns and API alternatives as `getCoalescedEvent` API introduced earlier.
+So in order to match it with `getCoalescedEvents` as well as exposing an ergonomic API we will use the same format as `getCoalescedEvents` by returning a list of `PointerEvent` objects.
+The events in the result list will have timestamps in the future and UA believes those events are going to be where the pointer is heading.
+This solution will match other platforms such as [prediction](https://developer.apple.com/documentation/uikit/touches_presses_and_gestures/handling_touches_in_your_view/minimizing_latency_with_predicted_touches) on iOS.
+You can fine examples on how to use the proposed API in the [proposed specification](https://w3c.github.io/pointerevents/extension.html#examples).
+
+
+## PointerRawUpdate
+Nowdays some UAs delay the events until before the next rAF as explained earlier in this document.
+For example Chromium delays all move events until right before rAF and Firefox does the same if the first event in the frame causes any changes to the DOM.
+However, there is another low latency need that this way of delaying events would break.
+
+### Problem
+There are some apps that may need to handle the events regardless of any visual effect.
+For example, some of the apps might be sending events over the network like in the cloud gaming use cases and some others might interact with audio devices.
+Either way delaying the delivery of the events would hurt the experience on these apps. So this problem needs a way of delivering the events without any delay.
```

Comment:
drop the "Either way". in the second sentence, I'd change "So this problem needs a way" to "The app needs a way"